R/tidyverse.R

```diff
@@ -23,9 +23,10 @@
 #'
 #' * `use_tidy_issue_template()`: creates a standard tidyverse issue template.
 #'
-#' @md
+#' * `use_tidy_support`: creates support resources document for GitHub repo
+#'    using SUPPORT.md.
+#'
 #' @name tidyverse
-NULL
```

Comment:
This `NULL` line needs to stay. This is the area of the merge conflict as well. Do you want to try to straighten it out in your branch/PR? If not, I will resolve.

Reply:
Thanks, Jenny. I _think_ I resolved them!